pages/flashbar/common.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { FlashbarProps } from '~components';
+
+export const i18nStrings = {
+  ariaLabel: 'Notifications',
+  notificationBarText: 'Notifications',
+  notificationBarAriaLabel: 'View all notifications',
+  errorIconAriaLabel: 'Error',
+  successIconAriaLabel: 'Success',
+  warningIconAriaLabel: 'Warning',
+  infoIconAriaLabel: 'Information',
+  inProgressIconAriaLabel: 'In progress',
+};
+
+const RANDOM_NUMBER_RANGE = 1000;
+
+export function generateItem(
+  type: FlashbarProps.Type,
+  dismiss: (index: string) => void,
+  hasHeader = false,
+  initial = false
+): FlashbarProps.MessageDefinition {
+  const randomKey = `key_${Math.floor(Math.random() * RANDOM_NUMBER_RANGE)}`;
```

Comment:
We have a util you can use: https://github.com/cloudscape-design/components/tree/main/src/internal/hooks/use-unique-id

Reply:
Follow-up micro-PR: https://github.com/cloudscape-design/components/pull/724